packages/core/admin/admin/src/components/GuidedTour/Homepage/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { useGuidedTour } from '@strapi/helper-plugin';
+import { useIntl } from 'react-intl';
+import { Stack } from '@strapi/design-system/Stack';
+import { Flex } from '@strapi/design-system/Flex';
+import { Box } from '@strapi/design-system/Box';
+import { Typography } from '@strapi/design-system/Typography';
+import { LinkButton } from '@strapi/design-system/LinkButton';
+import { Button } from '@strapi/design-system/Button';
+import ArrowRight from '@strapi/icons/ArrowRight';
+import StepperHomepage from './components/Stepper';
+import layout from '../layout';
+
+const GuidedTourHomepage = () => {
+  const { guidedTourState, setSkipped } = useGuidedTour();
+  const { formatMessage } = useIntl();
+
+  const sections = Object.entries(layout).map(([key, val]) => ({
+    key,
+    title: val.home.title,
+    content: (
+      <LinkButton to={val.home.cta.target} endIcon={<ArrowRight />}>
+        {formatMessage(val.home.cta.title)}
+      </LinkButton>
+    ),
+  }));
+
+  const enrichedSections = sections.map(section => ({
+    isDone: Object.entries(guidedTourState[section.key]).every(([, value]) => value),
+    ...section,
+  }));
+
+  const activeSection = enrichedSections.find(section => !section.isDone)?.key;
+
+  return (
+    <Box
+      hasRadius
+      shadow="tableShadow"
+      paddingTop={7}
+      paddingRight={4}
+      paddingLeft={7}
+      paddingBottom={4}
+      background="neutral0"
+    >
+      <Stack size={6}>
+        <Typography variant="beta" as="h2">
+          {formatMessage({
+            id: 'app.components.GuidedTour.title',
+            defaultMessage: '3 simple steps',
```

Comment:
Wondering about the copy here: as I user I'd ask: 3 simple steps to achieve what? Can we find something that explains a bit more, what the tour is about? ex: 3 steps to create your first content type

Also: in order to use inclusive language, we should avoid "simple" - it might be an easy task for one person, but a hard one for another ...

Reply:
Hi @gu-stav, thank you so much for your review, it is really interesting!
About all product copy / text change proposal, we will let @iparidhi and @maevalienard handle them if it's okay with you 😃

Reply:
I agree on the "simple" :) 

About the global title, I would recommend finding something short and consistent across the different screens. For instance, "3 steps to create your first content-type" is a big chunk, this would take a lot of space and is the first colored piece in the modal, I'm afraid this might be not tempting for the user to read the rest. 

I'd go for something like "3 steps to start" or so

Reply:
I think it should be looked at in the entirety of the welcome message. 
<img width="766" alt="Screenshot 2022-02-08 at 11 58 37 AM" src="https://user-images.githubusercontent.com/86970690/152974953-b1a8aa09-ce0f-4889-bc85-703b5a4b4033.png">

To keep it short yet coherent (and avoid repetition), my suggestion would be "3 steps to get started".

Reply:
@iparidhi I agree and think your suggestions sounds good. I'm not sure on the capitalization of the headline though, but I leave that up to you :)

Reply:
@iparidhi Since you were mentioning of looking at the entirety: the welcome message should maybe use `Content-Type` instead of `Content type` too?

Reply:
@gu-stav I replaced every Content-type / Single-type by Content type / Single type, it was a request from one of Jira ticket about guided tour!